### PR TITLE
modification to support downloads and cross account scripting

### PR DESCRIPTION
**Pull request by dunkgray**

### CODE REVIEW

File: datacube_alchemist/upload.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
 import os
 import tempfile
+import boto3
 from distutils.dir_util import copy_tree
 from pathlib import Path
+import mimetypes
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
```
Comment (Kirill888) [inline datacube_alchemist/upload.py]: this is python2 work-around, we don't do python2

File: datacube_alchemist/upload.py [hunk 2]
```diff
@@ -39,28 +46,35 @@ def upload_if_needed(self):
...
         """
         if self.upload is True:
-            self.upload_now()
+            self.upload_now_change_control()
 
-    def upload_now(self):
-        fs = fsspec.filesystem('s3')
-        # fs.put(self._location, self.s3location, recursive=True)
+    def upload_now_change_control(self):
+        s3_resource = boto3.resource('s3')
```
Comment (Kirill888) [inline datacube_alchemist/upload.py]: there is this in datacube-core now:

https://github.com/opendatacube/datacube-core/blob/develop/datacube/utils/aws/__init__.py#L327

includes handling of s3 urls, bu might not include ACLs, still would be nice to not duplicate code too much

File: datacube_alchemist/upload.py (hunk 1 above)
Comment (dunkgray) [reply datacube_alchemist/upload.py]: Thanks, I'll remove it

File: datacube_alchemist/upload.py (hunk 2 above)
Comment (dunkgray) [reply datacube_alchemist/upload.py]: I talked to Kirill.  Let's get this fix in now and refactor later.

**Pull request merged**